aten/src/ATen/native/mps/operations/ReduceOps.mm

```diff
@@ -545,6 +571,7 @@ void impl_func_norm_mps(
     }
 
     auto inputPlaceholder = native_mps::Placeholder();
+    auto otherPlaceholder = native_mps::Placeholder();
```

Comment:
```suggestion
    auto otherPlaceholder = native_mps::Placeholder(cachedGraph->otherTensor_, other_tensor);
```

Reply:
`other` is present only for `cdist`. In case of `norm.out` it doesn't represent anything.
I didn't want to create the placeholder in that case, since it might do some unneeded gather/blit operations even for `norm.out` when creating the `other` placeholder.